indexing-hadoop/pom.xml

```diff
@@ -121,6 +173,16 @@
                             <outputFile>
                                 ${project.build.directory}/${project.artifactId}-${project.version}-selfcontained.jar
                             </outputFile>
+                            <filters>
+                                <filter>
+                                    <artifact>*:*</artifact>
+                                    <excludes>
+                                        <exclude>META-INF/*.SF</exclude>
```

Comment:
It was weirding out when I was using hadoop self contained. But that should be in a different PR. Removing